### DISCUSSION TITLE
In Greek mythology, a giant Cyclops had how many eyes?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#137] [Galaxy Trivia] [answered]
**Hint:** Mike Wazowski

**Comment by Tynab** [comment, marked as answer]
In Greek mythology, a giant Cyclops is typically depicted as having only one eye in the middle of its forehead. The Cyclops were one-eyed giants who were known for their strength and craftsmanship. They were said to be the sons of Uranus and Gaea, and they were often portrayed as either friendly or hostile towards humans, depending on the particular myth. The most famous Cyclops in Greek mythology is probably Polyphemus, who appears in Homer's epic poem, the Odyssey.
